test/optimize-deps/test/ssr.test.ts

```diff
@@ -0,0 +1,10 @@
+// @vitest-environment node
+
+import { expect, test } from 'vitest'
+
+// @ts-expect-error untyped
+import { importMetaUrl } from '@vitest/test-dep-url'
+
+test('import.meta.url', () => {
+  expect(importMetaUrl).toContain('/node_modules/.vitest/deps_ssr/')
```

Comment:
Interestingly, ssr.optimizeDeps seems to be failing only on Windows.
Unfortunately `AssertionError` diff doesn't print everything...

Reply:
Shouldn't this be `\\node_modules\\.vitest\\deps_ssr`

Reply:
It's `import.meta.url`, so I think path is already normalized with `file://something/node_modules/...`. The diff is truncated but you can see the first part here https://github.com/vitest-dev/vitest/actions/runs/7927404820/job/21643736718?pr=5215#step:8:697

Reply:
Hmm, the same test passed on next run (though Windows failed on a different test) https://github.com/vitest-dev/vitest/actions/runs/7927585760/job/21644224823?pr=5215, so maybe this is the flakiness of optimizeDeps.

I'll retry a few times to see what happens.

Reply:
The flakiness is the reason why we disabled optimizer for now

Reply:
It looks like only windows + ssr combination is flaky, so I only targeted this to disable.
Later I'll look into further if there's something sketchy with other cases.